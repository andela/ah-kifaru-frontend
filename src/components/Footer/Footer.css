@import url('https://fonts.googleapis.com/css?family=Poppins:400,500,600,700,800,900|Roboto&display=swap');


  .footer {
    font-family: 'Poppins', sans-serif;
    position: fixed;
    left: 0px;
    bottom: 0px;
  }
