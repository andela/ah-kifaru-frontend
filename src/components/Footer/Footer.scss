@import url('https://fonts.googleapis.com/css?family=Poppins:400,500,600,700,800,900|Roboto&display=swap');
@import '../../../public/styles/colors.scss';

.footer {
  font-family: 'Poppins', sans-serif;
  position: relative;
  left: 0px;
  bottom: 0px;
  background: #ffffff;
  border-top: 1px solid $light-grey;
  @apply text-danger text-base font-bold py-10;
}
