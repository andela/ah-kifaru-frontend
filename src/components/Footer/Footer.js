import React from 'react';
import './Footer.css';
import YoutubeIcon from '../../assets/icons/youtube';
import FacebookIcon from '../../assets/icons/facebook';
import LinkedInIcon from '../../assets/icons/linkedin';
import InstagramIcon from '../../assets/icons/instagram';
import TwitterIcon from '../../assets/icons/twitter';

const Footer = () => {
  return (
    <footer className="footer flex flex-wrap w-full justify-center px-5">
      <div className="sm:w-1/2 h-8">
        <p className=" flex justify-start">Designed by Kifaru Team</p>
      </div>
      <div className="sm:w-1/2 h-8">
        <ul className="flex justify-end items-center">
          <li className="mx-1 px-2 py-1 cursor-pointer">
            <TwitterIcon />
          </li>
          <li className="mx-1 px-2 py-1 cursor-pointer">
            <FacebookIcon />
          </li>
          <li className="mx-1 px-2 py-1 cursor-pointer">
            <InstagramIcon />
          </li>
          <li className="mx-1 px-2 py-1 cursor-pointer">
            <YoutubeIcon />
          </li>
          <li className="mx-1 px-2 py-1 cursor-pointer">
            <LinkedInIcon />
          </li>
        </ul>
      </div>
    </footer>
  );
};

export default Footer;
