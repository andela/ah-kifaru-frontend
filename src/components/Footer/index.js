import React from 'react';
import './Footer.scss';
import {
  YoutubeIcon,
  FacebookIcon,
  LinkedInIcon,
  InstagramIcon,
  TwitterIcon
} from '../../assets/icons';

const Footer = () => (
  <footer className="footer flex flex-wrap w-full justify-center px-10 sticky">
    <div className="md:w-1/2 h-8">
      <p className=" flex justify-start">Designed by Kifaru Team</p>
    </div>
    <div className="md:w-1/2 h-8">
      <ul className="flex justify-end items-center">
        <li className="mx-1 px-2 py-1 cursor-pointer">
          <TwitterIcon />
        </li>
        <li className="mx-1 px-4 py-1 cursor-pointer">
          <FacebookIcon />
        </li>
        <li className="mx-1 px-4 py-1 cursor-pointer">
          <InstagramIcon />
        </li>
        <li className="mx-1 px-4 py-1 cursor-pointer">
          <YoutubeIcon />
        </li>
        <li className="mx-1 px-4 py-1 cursor-pointer">
          <LinkedInIcon />
        </li>
      </ul>
    </div>
  </footer>
);
export default Footer;
