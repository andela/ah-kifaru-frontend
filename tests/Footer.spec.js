import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from './enzyme';
import Footer from '../src/components/Footer/Footer';

describe('Footer layout', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<Footer />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
