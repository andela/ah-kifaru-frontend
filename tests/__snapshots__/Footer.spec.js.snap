// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer layout should render without crashing 1`] = `
<footer
  className="footer flex flex-wrap w-full justify-center px-5"
>
  <div
    className="sm:w-1/2 h-8"
  >
    <p
      className=" flex justify-start"
    >
      Designed by Kifaru Team
    </p>
  </div>
  <div
    className="sm:w-1/2 h-8"
  >
    <ul
      className="flex justify-end items-center"
    >
      <li
        className="mx-1 px-2 py-1 cursor-pointer"
      >
        <[object Object] />
      </li>
      <li
        className="mx-1 px-2 py-1 cursor-pointer"
      >
        <[object Object] />
      </li>
      <li
        className="mx-1 px-2 py-1 cursor-pointer"
      >
        <[object Object] />
      </li>
      <li
        className="mx-1 px-2 py-1 cursor-pointer"
      >
        <[object Object] />
      </li>
      <li
        className="mx-1 px-2 py-1 cursor-pointer"
      >
        <[object Object] />
      </li>
    </ul>
  </div>
</footer>
`;
